dice/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@ add_library(dice ${SRCFILES})
 
 add_dependencies(dice dshr streams)
 target_include_directories (dice PRIVATE ${ESMF_F90COMPILEPATHS})
-target_include_directories (dice PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../fox/include")
```

Comment:
I believe that these are still needed.   Some compilers need access to modules referenced indirectly by the current source file.  I think nag and pgi both demonstrate this issue.

Reply:
I tested without it and it was working but I was using Intel. So, if you thing we need those, I could include them easily.

Reply:
I add them again to solve the issue under NAG